src/model/modelUtil.ts

```diff
@@ -18,3 +21,18 @@ export function makeCommunicationRequests(patient: Patient, planDefinition: Plan
 export function makeCarePlan(planDefinition: PlanDefinition, patient: Patient, communicationRequests: CommunicationRequest[]): CarePlan {
     return CarePlan.createIsaccCarePlan(patient, planDefinition, communicationRequests);
 }
+
+export function unmarkTestPatient(
+  client: Client,
+  patientId: string
+): Promise<any> {
+  if (!client) return null;
+  return client.request({
+    url: `Patient/${patientId}/$meta-delete`,
+    method: "POST",
+    headers: {
+      "content-type": "application/xml",
+    },
+    body: `<Parameters xmlns="http://hl7.org/fhir"><parameter><name value="meta"/><valueMeta><security><system value="${SystemURL.testPatientUrl}"/><code value="${Patient.TEST_PATIENT_SECURITY_CODE}"/></security></valueMeta></parameter></Parameters>`,
```

Comment:
I've been messing around with Parameter FHIR resources from Bulk FHIR work, and I think this JSON should be equivalent to the XML [from the example](https://www.hl7.org/fhir/resource-operation-meta-delete.html#examples)

I was able to successfully test the JSON with curl as you and Paul had (thanks for the notes!), but haven't tested the suggested changes in this PR.
```
RESOURCE='{"resourceType":"Parameters","parameter":[{"name":"meta","valueMeta":{"security":[{"system":"http://terminology.hl7.org/CodeSystem/v3-ActReason","code":"HTEST"}]}}]}'

curl -X POST -H 'Content-Type: application/json' -d "$RESOURCE" 'https://fhir.isacc.dev.cirg.uw.edu/fhir/Patient/2/$meta-delete'

{
  "resourceType": "Parameters",
  "parameter": [ {
    "name": "return",
    "valueMeta": {
      "versionId": "137",
      "lastUpdated": "2023-08-02T16:52:58.841+00:00"
    }
  } ]
}
```

```suggestion
    headers: {
      "content-type": "application/json",
    },
    body: {
      "resourceType" : "Parameters",
      "parameter" : [{
          "name" : "meta",
          "valueMeta" : {
            "security": [{
              "system": SystemURL.testPatientUrl,
              "code": Patient.TEST_PATIENT_SECURITY_CODE
            }]
          }
      }]
    },
```

Reply:
thank you @ivan-c! I verified that the JSON version works as well and documented in the code.